packages/augur-ui/src/modules/create-market/saved-drafts.tsx

```diff
@@ -69,12 +72,13 @@ export default class SavedDrafts extends React.Component<
         normalOnMobile
         content={
           <div className={Styles.SavedDrafts}>
-            {draftsSorted.map(key => 
-              <DraftRow key={key} draft={drafts[key]} removeDraft={removeDraft} updateNewMarket={updateNewMarket} updatePage={updatePage} />
+            {draftsSorted.map(key =>
+              <DraftRow key={key} draft={drafts[key]} removeDraft={removeDraft} updateNewMarket={updateNewMarket}
```

Comment:
I think prettier would have caught this, but can we get this as a list style instead of a longer component:
```
<DraftRow
  key={key}
  draft={drafts[key]}
  removeDraft={removeDraft}
  updateNewMarket={updateNewMarket}
  updatePage={updatePage}
/>
```